### ISSUE TITLE
Correlation viewer doesn't show single expocode very well

### ISSUE THREAD

**Issue by karlmsmith** [NOAA-PMEL/LAS#1610]
**Reported by @kevin-obrien on 29 Jul 2014 23:05 UTC**

In the correlation viewer and looking again at fco2 as a func of time colored by expocode and I isoloate a single expocode by clicking one of the icon buttons and get a plot that has the values plotted directly on the vertical axes.  Here's the url:

```
http://dunkel.pmel.noaa.gov:8680/socat/productserver/templates/SPPV.vm?catid=socatV3_c6c1_d431_8194&xml=%3C%3Fxml+version%3D%221.0%22%3F%3E%3ClasRequest+href%3D%22file%3Alas.xml%22%3E%3Cargs%3E%3Clink+match%3D%22%2Flasdata%2Fdatasets%2FsocatV3_c6c1_d431_8194%2Fvariables%2FfCO2_recommended-socatV3_c6c1_d431_8194%22+%2F%3E%3Cregion%3E%3Crange+type%3D%22x%22+low%3D%22-70.78569082774%22+high%3D%22-70.233010357942%22+%2F%3E%3Crange+type%3D%22y%22+low%3D%2240.500771669691%22+high%3D%2240.719698421053%22+%2F%3E%3Crange+type%3D%22t%22+low%3D%2212-Aug-2012+00%3A00%22+high%3D%2201-Mar-2014+00%3A00%22+%2F%3E%3C%2Fregion%3E%3Cconstraint+type%3D%22variable%22+id%3D%22socatV3_c6c1_d431_8194_fCO2_recommended-socatV3_c6c1_d431_8194%22+op%3D%22ne%22%3E%3Clink+match%3D%22%2Flasdata%2Fdatasets%2FsocatV3_c6c1_d431_8194%2Fvariables%2FfCO2_recommended-socatV3_c6c1_d431_8194%22+%2F%3E%3Cv%3ENaN%3C%2Fv%3E%3C%2Fconstraint%3E%3C%2Fargs%3E%3Clink+match%3D%22%2Flasdata%2Foperations%2Foperation[%40ID%3D%27SPPV%27]%22+%2F%3E%3Cproperties%3E%3Cferret%3E%3Cview%3Exyt%3C%2Fview%3E%3Csize%3E.8333%3C%2Fsize%3E%3Cimage_format%3Egif%3C%2Fimage_format%3E%3Cannotations%3Efile%3C%2Fannotations%3E%3C%2Fferret%3E%3Clas%3E%3Coutput_type%3Ehtml%3C%2Foutput_type%3E%3Cdebug%3Efalse%3C%2Fdebug%3E%3C%2Flas%3E%3Cproduct_server%3E%3Cversion%3E7.3%3C%2Fversion%3E%3C%2Fproduct_server%3E%3C%2Fproperties%3E%3C%2FlasRequest%3E#%3ClasRequest%20href=%22file:las.xml%22%3E%3Cargs%3E%3Clink%20match=%22/lasdata/datasets/socatV3_c6c1_d431_8194/variables/time-socatV3_c6c1_d431_8194%22/%3E%3Clink%20match=%22/lasdata/datasets/socatV3_c6c1_d431_8194/variables/fCO2_recommended-socatV3_c6c1_d431_8194%22/%3E%3Clink%20match=%22/lasdata/datasets/socatV3_c6c1_d431_8194/variables/expocode-socatV3_c6c1_d431_8194%22/%3E%3Cregion%3E%3Crange%20type=%22t%22%20low=%2212-Aug-2012%2000:00%22%20high=%2201-Mar-2014%2000:00%22/%3E%3Crange%20type=%22x%22%20low=%22-70.78569082774%22%20high=%22-70.233010357942%22/%3E%3Crange%20type=%22y%22%20low=%2240.500771669691%22%20high=%2240.719698421053%22/%3E%3C/region%3E%3Cconstraint%20type=%22variable%22%20id=%22socatV3_c6c1_d431_8194_fCO2_recommended-socatV3_c6c1_d431_8194%22%20op=%22ne%22%3E%3Clink%20match=%22/lasdata/datasets/socatV3_c6c1_d431_8194/variables/fCO2_recommended-socatV3_c6c1_d431_8194%22/%3E%3Cv%3ENaN%3C/v%3E%3C/constraint%3E%3Cconstraint%20type=%22variable%22%20id=%22socatV3_c6c1_d431_8194_fCO2_recommended-socatV3_c6c1_d431_8194%22%20op=%22ne%22%3E%3Clink%20match=%22/lasdata/datasets/socatV3_c6c1_d431_8194/variables/fCO2_recommended-socatV3_c6c1_d431_8194%22/%3E%3Cv%3ENaN%3C/v%3E%3C/constraint%3E%3C/args%3E%3Clink%20match=%22/lasdata/operations/operation[@ID=%27Trajectory_correlation_plot%27]%22/%3E%3Cproperties%3E%3Cferret%3E%3Cview%3Exyt%3C/view%3E%3Csize%3E.8333%3C/size%3E%3Cimage_format%3Egif%3C/image_format%3E%3Cannotations%3Efile%3C/annotations%3E%3Ccruise_list%3E33HH20130701,%3C/cruise_list%3E%3C/ferret%3E%3Clas%3E%3Coutput_type%3Exml%3C/output_type%3E%3Cdebug%3Efalse%3C/debug%3E%3C/las%3E%3Cproduct_server%3E%3Cversion%3E7.3%3C/version%3E%3Cui_timeout%3E20%3C/ui_timeout%3E%3Cuse_cache%3Etrue%3C/use_cache%3E%3C/product_server%3E%3Cdata%3E%3Ccount%3E3%3C/count%3E%3C/data%3E%3Cdata_0%3E%3Curl%3E/home/users/kobrien/armstrong/tomcat/webapps/socat/output/FCEF5F43D8D535E293D34E91F7CD4ED4_netcdf.nc%3C/url%3E%3C/data_0%3E%3Cdata_1%3E%3Curl%3E/home/users/kobrien/armstrong/tomcat/webapps/socat/output/FCEF5F43D8D535E293D34E91F7CD4ED4_netcdf.nc%3C/url%3E%3C/data_1%3E%3Cdata_2%3E%3Curl%3E/home/users/kobrien/armstrong/tomcat/webapps/socat/output/FCEF5F43D8D535E293D34E91F7CD4ED4_netcdf.nc%3C/url%3E%3C/data_2%3E%3C/properties%3E%3C/lasRequest%3E
```

and I've attached the plot for reference.

Migrated-From: http://dunkel.pmel.noaa.gov/trac/las/ticket/1603

**Comment by karlmsmith** [comment]
**Attachment from @kevin-obrien on 29 Jul 2014 23:05 UTC**

REPLACE THIS TEXT WITH UPLOADED FILE /home/merlot/ksmith/git/trac2github/attachments/TRAC_1603_GIT_1610/LAS_correlation_single_expo_fail.gif

**Comment by karlmsmith** [comment]
**Attachment from @AnsleyManke on 30 Jul 2014 17:31 UTC**
prop-prop plots on time axis; make sure data not drawn at time-axis edges
REPLACE THIS TEXT WITH UPLOADED FILE /home/merlot/ksmith/git/trac2github/attachments/TRAC_1603_GIT_1610/1603.gif

**Comment by karlmsmith** [comment]
**Modified by @kevin-obrien on 29 Jul 2014 23:06 UTC**

**Comment by karlmsmith** [comment]
**Comment by @AnsleyManke on 30 Jul 2014 17:32 UTC**
Together with the fix for #1602, make sure there is some padding at the ends of time axes when plotting variable vs time in the prop-prop viewer.
